internal/plugin/connectors/tcp/mssql/connector.go

```diff
@@ -96,13 +98,19 @@ Overview of the connection process
 type SingleUseConnector struct {
 	backendConn net.Conn
 	clientConn  net.Conn
+	clientBuff  *mssql.TdsBuffer
 	logger      log.Logger
 }
 
 // https://docs.microsoft.com/en-us/sql/database-engine/configure-windows/configure-the-network-packet-size-server-configuration-option
 // Default packet size remains at 4096 bytes
 const bufferSize uint16 = 4096
 
+type connectResult struct {
+	conn *mssql.Conn
+	err error
+}
+
 // Connect implements the tcp.Connector func signature
 //
 // It is the main method of the SingleUseConnector. It:
```

Comment:
We should update this class comment with any new steps

Reply:
Adding to Bradley's comment, the sequence diagram should also be updated to show login being read from client and (real) login response being sent back to client. (https://github.com/cyberark/secretless-broker/pull/1050/files#diff-a4bfca951ffb2d0c843eb664663a402dL71)